### PR TITLE
Culling fixes

### PR DESCRIPTION
**Pull request by hyperlogic**
This fixes two somewhat related issues.

1. Avatars in the corner of your view will freeze, and not update joints or position.  This bug has always been there, but is more apparent with some of our new avatars.  Basically we weren't computing the bounding sphere correctly.

2. Entities can sometimes flicker when you move your head.  This bug was introduced by #7487.  We need to properly update the renderItem when it is first added to the scene.

### CODE REVIEW

**Comment by hyperlogic** [comment]
https://app.asana.com/0/45663370065758/107022902619927
https://app.asana.com/0/45663370065758/107051724889235

**Comment by hifi-gustavo** [comment]
Build console output for commit: bc967f0ab0d12fc69b44575d0461ec1eb3661231 available at: https://gist.github.com/3359fc46a0576baba20b728d95575010<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7530.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7530.dmg

**Comment by ZappoMan** [comment]
Let's make sure we get a before and after FPS test on this...

**Comment by hifi-gustavo** [comment]
Build console output for commit: d670853941c8ec0d2b39a9fdfdb1923658729ef6 available at: https://gist.github.com/3359fc46a0576baba20b728d95575010<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7530.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7530.dmg

**Comment by ZappoMan** [comment]
needs update from master

**Comment by hifi-gustavo** [comment]
Build console output for commit: 5e81cccc6cecf717d674d1fd384dd8ce3a3898b0 available at: https://gist.github.com/3359fc46a0576baba20b728d95575010<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7530.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7530.dmg

**Comment by hyperlogic** [comment]
Looks like the build failed to download sixense...  Test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: 5e81cccc6cecf717d674d1fd384dd8ce3a3898b0 available at: https://gist.github.com/dd7a843a343af9767324291f8c1c460e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7530.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7530.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 75219573a4a7791f8feb546beff28b2436ec923d available at: https://gist.github.com/3359fc46a0576baba20b728d95575010<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7530.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7530.dmg

File: libraries/render-utils/src/AnimDebugDraw.cpp [hunk 1]
```diff
@@ -392,20 +392,14 @@ void AnimDebugDraw::update() {
 
         assert(numVerts == (v - verts));
 
-        // The RenderItem culling is not working correctly
-        // Workaround this issue by using the default constructed
-        // item._bound which is a 16 km cube.
-        /*
```
Comment (ZappoMan) [inline libraries/render-utils/src/AnimDebugDraw.cpp]: did you mean to remove this "commented out code"???

Comment (ZappoMan) [inline libraries/render-utils/src/AnimDebugDraw.cpp]: in other words -- you've removed the comments that made this loop of code not get executed... was that intentional?

Comment (hyperlogic) [inline libraries/render-utils/src/AnimDebugDraw.cpp]: Yep, that code is A-OK now that AABox operator+= works.

**Comment by hyperlogic** [comment]
This is admittedly difficult to reproduce, but the forum had the some of the biggest issues.

1. If you stand in the main forum room, and look toward the patio with the whiteboard behind you.  The base of the fish-tank on your left can sometimes become invisible as you turn your head side to side.  You can set a bookmark here, and jump to it from another location and this will sometimes cause it to reproduce more often.

2. If you go to the changing room, also in forum,  and place one or more models on the pedestal.  Sometimes the dolls on the shelf will have parts of them that do not render, specifically the two female models Kate and Art3mis.

**Comment by cozza13** [comment]
@thoys is testing this one

**Comment by thoys** [comment]
## Testing results

Unfortunately I wasn't able to reproduce the issues (1. + 2.) in the master version (Teleported back and forward a few times). But the smoke test went well in the PR. Everything shows up correctly, tried moving my head very quickly with the DK2 a few times aswell to see if things are visible half and they did appear normal aswell.
:+1: :heavy_check_mark: QA successful. Have a good weekend!

**Pull request merged**